api/aperture/policy/language/v1/fluxmeter.proto

```diff
@@ -14,15 +14,26 @@ import "protoc-gen-openapiv2/options/annotations.proto";
 //
 // :::
 //
-// Example of a selector that creates a histogram metric for all HTTP requests
-// to particular service:
+// Example:
 // ```yaml
-// selector:
-//   service_selector:
-//     service: myservice.mynamespace.svc.cluster.local
-//   flow_selector:
-//     control_point: ingress
+// flow_selector:
+//  static_buckets:
```

Comment:
again, not correct, static_buckets are not a part of flow_selector
and you should only show one of all those buckets definitons

Reply:
okay, about buckets you should show all if they were shown before

Reply:
Corrected in next pr

Reply:
there were not buckets in the previous one, I'll show only one